### PR TITLE
Move validation of jurisdiction into Jurisdiction model

### PR DESCRIPTION
**Pull request by linusnorton**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java [hunk 1]
```diff
@@ -797,4 +746,12 @@ public void addToDocumentsGenerated(final ListValue<DivorceDocument> listValue)
...
         }
     }
+
+    public Jurisdiction getJurisdiction() {
+        if (jurisdiction == null) {
+            jurisdiction = new Jurisdiction();
+        }
+
+        return jurisdiction;
+    }
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java:756]: do we need this ?

**Review by nitinprabhu — COMMENTED** [review]
should we bump the case type as the field names have been changed ?

**Comment by linusnorton** [comment]
> should we bump the case type as the field names have been changed ?

I will double check against the frontend but they should be the same

File: src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java (hunk 1 above)
Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java:756]: I tried to set a default in the property definition but it causes issues with lombok so this was the only way I could see if ensuring there was an object

**Review by nitinprabhu — APPROVED** [review]

**Pull request merged**